roles/deploy_harbor/tasks/main.yaml

```diff
@@ -0,0 +1,47 @@
+
+# - name: Create Secret for Redis
+#   kubernetes.core.k8s:
+#     state: present
+#     definition: 
+#       apiVersion: v1
+#       kind: Secret
+#       type: Opaque             
+#       metadata:
+#         name: "{{ secret_name }}"
+#         namespace: "{{ harbor_namespace }}"     
+#       data:
+#         username: "{{ 'REDIS_PASSWORD' | b64encode }}"
+#         password: "{{ lookup('env', 'REDIS_PASSWORD') | b64encode }}"

```

Comment:
Can we re-enable this in the next PR and check it works using the secret instead of a password in values.yml?